components/form/style/animation.less

```
@@ -0,0 +1,38 @@
+@import (reference) '../../style/themes/index';
+
+// ================================================================
+// =                    Form Validate Animation                   =
+// ================================================================
+
+@form-prefix-cls: ~'@{ant-prefix}-form';
+
+@animation-duration: 0.3s;
+@animate-distance: -5px;
+
+.animation(@type) {
+  &-@{type} {
+    animation: ~"@{type}" @animation-duration ease-in-out;
```

use Less variables instead


```suggestion
    animation: ~"@{type}" @animation-duration @ease-in-out;
```